apps/snitch_core/lib/core/data/model/payment/payment.ex

```diff
@@ -62,4 +62,22 @@ defmodule Snitch.Data.Model.Payment do
   def to_subtype(%Payment{payment_type: "ccd"} = payment) do
     CardPaymentModel.from_payment(payment.id)
   end
+
+  defmacro to_char(field, format) do
+    quote do
+      fragment("to_char(?, ?)", unquote(field), unquote(format))
+    end
+  end
+
+  def get_payment_count_by_date(start_date, end_date) do
+    Payment
+    |> where([p], p.inserted_at >= ^start_date and p.inserted_at <= ^end_date)
+    |> group_by([p], to_char(p.inserted_at, "dd Mon YYYY"))
```

Comment:
"dd Mon YYYY" should be uniform in my opinion `"DD MM YYYY"` what do you think? @anantanant2015